spec/components/table_row_component_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe TableRowComponent, type: :component do
+  pending "add some examples to (or delete) #{__FILE__}"
+
```

Comment:
Great job 👏 @niladrix719! Let's add some tests for this, by including some dummy text and checking if the rendered component contains that text.